docs/index.md

```diff
@@ -18,17 +18,3 @@ The Jenkins-k8s Operator is a member of the Ubuntu family. It's an open source p
 - [Contribute](Contribute)
 
 Thinking about using the Jenkins-k8s Operator for your next project? [Get in touch](https://chat.charmhub.io/charmhub/channels/charm-dev)!
-
```

Comment:
This file doesn't match the new standards. I miss the "Contributing to this documentation" and "n this documentation" sections